### ISSUE TITLE
Should no restart vpn on configuration changes

### ISSUE THREAD

**Issue by amz** [cad/ovpm#85]
Should not restart vpn on adding network, adding users, associate/dissociate user to/from network.

**Comment by cad** [comment]
Agreed! It’s already on my list but I need to switch to OpenVPN management interface in order to manage the underlying OpenVPN server dynamically. Yet leaving this open.

**Comment by fregie** [comment]
Any update of this?
Maybe I can help implement this feature
@cad
